### PR TITLE
Implement SES resources types

### PR DESCRIPTION
**Pull request by hongil0316**
## Description

Fixes https://github.com/gruntwork-io/cloud-nuke/issues/447.

## TODOs

Read the [Gruntwork contribution guidelines](https://gruntwork.notion.site/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e).

- [ ] Update the docs.
- [ ] Run the relevant tests successfully, including pre-commit checks.
- [ ] Ensure any 3rd party code adheres with our [license policy](https://www.notion.so/gruntwork/Gruntwork-licenses-and-open-source-usage-policy-f7dece1f780341c7b69c1763f22b1378) or delete this line if its not applicable.
- [ ] Include release notes. If this PR is backward incompatible, include a migration guide.
- [ ] Attention Grunts - if this PR adds support for a new resource, ensure the `nuke_sandbox` and `nuke_phxdevops` jobs in `.circleci/config.yml` have been updated with appropriate exclusions (either directly in the job or via the `.circleci/nuke_config.yml` file) to prevent nuking IAM roles, groups, resources, etc that are important for the test accounts.

## Release Notes (draft)

Added / Removed / Updated [X].

### CODE REVIEW

File: aws/resources/ses_configuration_set.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	for _, set := range sets {
+		_, err := s.Client.DeleteConfigurationSet(&ses.DeleteConfigurationSetInput{
+			ConfigurationSetName: set,
+		})
+
+		// Record status of this resource
+		e := report.Entry{
+			Identifier:   aws.StringValue(set),
+			ResourceType: "SES configuartion set",
```
Comment (denis256) [inline aws/resources/ses_configuration_set.go:52]: configuartion

File: aws/resources/ses_configuration_set.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/report"
+	"github.com/gruntwork-io/go-commons/errors"
+)
+
+// Returns a formatted string of ses-configuartion set names
+func (s *SesConfigurationSet) getAll(c context.Context, configObj config.Config) ([]*string, error) {
+	// Remove defalt route table, that will be deleted along with its TransitGateway
```
Comment (denis256) [inline aws/resources/ses_configuration_set.go:16]: `default`

Comment (hongil0316) [reply aws/resources/ses_configuration_set.go]: Updated.

File: aws/resources/ses_configuration_set.go (hunk 1 above)
Comment (hongil0316) [reply aws/resources/ses_configuration_set.go]: Updated.

**Review by denis256 — APPROVED** [review]

**Pull request merged**